src/network/bmproto.py

```diff
@@ -318,8 +318,8 @@ def decode_simple(self, char="v"):  # pylint: disable=inconsistent-return-statem
 
     def bm_command_error(self):
         """Decode an error message and log it"""
-        fatalStatus, banTime, inventoryVector, errorText = \
-            self.decode_payload_content("vvlsls")
+        fatalStatus, errorText = \
+            self.decode_payload_content("vvlsls")[0], self.decode_payload_content("vvlsls")[3]
```

Comment:
Perhaps I would add a comment describing elements 1 and 2 for documentation purposes, as the error command isn't documented well.

Reply:
this change has few sense and may cause problems, better:
```python
        err_values = self.decode_payload_content("vvlsls")
        fatalStatus = err_values[0]
        errorText = err_values[3]
```